platform/00-kubernetes-minikube-setup.sh

```diff
@@ -16,11 +16,39 @@ HELM_FILE_NAME=helm-v3.7.1-linux-amd64.tar.gz
 HELM_URL=https://get.helm.sh/$HELM_FILE_NAME
 HELM_SHA256=6cd6cad4b97e10c33c978ff3ac97bb42b68f79766f1d2284cfd62ec04cd177f4
 
-TKN_VERSION=0.21.0
+TKN_VERSION=v0.21.0
 TKN_FILE_NAME=tkn_0.21.0_Linux_x86_64.tar.gz
-TKN_URL=https://github.com/tektoncd/cli/releases/download/v$TKN_VERSION/$TKN_FILE_NAME
+TKN_URL=https://github.com/tektoncd/cli/releases/download/$TKN_VERSION/$TKN_FILE_NAME
 TKN_SHA256=2158a202e4b04ff73e6427b565355c7bfc8cbe16dc7058a0414fb16e7b97008c
 
+KUBECTL_VERSION=v1.22.3
+KUBECTL_FILE_NAME=kubectl
+KUBECTL_URL=https://dl.k8s.io/release/$KUBECTL_VERSION/bin/linux/amd64/kubectl
```

Comment:
Maybe this should be defined in the Linux install section since it is hard-coding the architecture `amd64`.